community/0-welcome.mdx

```diff
@@ -18,12 +18,12 @@ Or why not join our [GitHub Discussions community](https://github.com/microsoft/
 
 ## Contributing
 
-Check out or [contribuing guide](https://github.com/microsoft/playwright/blob/main/CONTRIBUTING.md) if you would like to contribute to Playwright.
+Check out our [contributing guide](https://github.com/microsoft/playwright/blob/main/CONTRIBUTING.md) if you would like to contribute to Playwright.
 
 
 ## Community Slack
 
-Join our community [Slack Channel](https://playwright.slack.com/join/shared_invite/zt-1b2uhuir8-wdq_gX1bjvrzgwdzaiAIqQ#/shared-invite/email) to connect with other developers using Playwright.
+Join our community [Slack Channel](https://aka.ms/playwright-slack) to connect with other developers using Playwright.
```

Comment:
The current link brings us to the following page.
I updated the slack invite, with the one from the sidebar. 

![image](https://user-images.githubusercontent.com/28659384/179446165-fbdf3337-db58-4c93-9ee5-f1a1125b64c3.png)